build_msvc/bitcoin_config.h.in

```diff
@@ -150,6 +150,9 @@
 /* ::wsystem is available */
 #define HAVE_SYSTEM 1
 
+/* Assume that an MSVC build will get dependencies from vcpk and thus use the latest version of libevent */
```

Comment:
Is there a chance that this is not the case in any situation ?

Reply:
I don't know. I would prefer a solution more in line with how this was solved in automake builds but I don't have time to learn enough about msbuild to implement it and it is currently broken in the default case.

Reply:
In fact the build in CI is now failing because it expects the other function signature.

`C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\src\httpserver.cpp(635,56): error C2664: 'void evhttp_connection_get_peer(evhttp_connection *,char **,uint16_t *)': cannot convert argument 2 from 'const char **' to 'char **' [C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\build_msvc\libbitcoin_node\libbitcoin_node.vcxproj]`

Reply:
Using the latest version of libevent in CI is now fixes the build.